deprecate.dd

```diff
@@ -10,6 +10,7 @@ $(SPEC_S Deprecated Features,
 
     $(TABLE2 Deprecated Features,
         $(THEAD Feature,                                                          Spec,  Dep,    Error,  Gone)
+        $(TROW $(DEPLINK Class allocators and deallocators),                      ?,      ?,     &nbsp;, &nbsp;)
```

Comment:
Similar to `delete` a couple of lines below, this should read `future, &nbsp;, &nbsp;, &nbsp;)` as this feature was neither removed from the spec (just a [note](https://dlang.org/spec/class.html#allocators) saying that it is "deprecated in D2"), nor the compiler emits any warnings if you use them.

Apart from that LGTM, we need to guide the users to better ways of using D.